appeals/web/src/server/appeals/appeals.types.d.ts

```diff
@@ -1,4 +1,5 @@
 import { Folder } from '@pins/appeals/index.js';
```

Comment:
I think these two definitions are wrong, but I would leave them for the moment as TsCheck is not complaining and we should really agree on how we're sharing definitions between API and WEB.